src/js/stores/transforms/AppFormFieldToModelTransforms.js

```diff
@@ -147,6 +147,14 @@ const AppFormFieldToModelTransforms = {
         }
         delete definition.consecutiveKey;
         delete definition.isRandomPort;
+        if (definition.vip != null) {
```

Comment:
Yes, `vip` is only used as a "field".
I have to do this magic, because somebody decided to introduce a field (the VIP-field) that is actually not covered by the real structure of the data sent to the server, so I need to transform this and can't to a 1:1 transformation.